src/components/ProgressBar/ProgressBar.tsx

```diff
@@ -0,0 +1,42 @@
+import { LinearProgress, Typography } from "@mui/material";
+import Box from "@mui/system/Box";
+import { useEffect, useState } from "react";
+
+interface ProgressBarProps {
+    children: React.ReactNode;
+}
+
+interface PackageDistribution {
+  id: number
+  name: string
+  address: string
+  postcode: string
+  city: string
+  country: string
+  lat: string
+  lng: string
+}

```

Comment:
It is a good idea to keep types in a separate file next to the component. Could you please move those declarations to types.ts file?